writing/automation.md

```diff
@@ -0,0 +1,27 @@
+# Automate Everything
+
+---
+
+- Time is valuable, don't do things computers can do for you. Remembering (calendar, checklists, reminders). Free yourself for more creative thinking and enjoying life
+- As programmer, goal is to build scripts so you don't have to do anything
+  - Automating yourself out of a job
+- Bringing into life
+
+  - Checklists and reminders through Butler
+    - No good notifications though
+    - Butler
+      - When due date, move into inbox
+      - When due date set, move into snoozed
+      - Regular
+        - Daily
+        - Weekly (mon, tue, fri, sun)
+        - Monthly
+  - Other things that may be useful to you
+    - IFTTT
+    - Google Scripts
+    - Personal Assistants
+    - Full on programming
+
+- Have a strong emotional tone
```

Comment:
Ok, I've figured out how to comment like this, but can't figure out how to do it inline like the other one above.